docs/content/preview/manage/backup-restore/lightweight-db-clone.md

```diff
@@ -0,0 +1,261 @@
+```
+
+{{<note title="Note">}}
+Ensure to not override your default host based authentication rules when trusting the local connection. You may need to add additional authentication lines to `ysql_hba_conf_csv` based on your specific configuration. For more information, see [host based authentication](../../../secure/authentication/host-based-authentication/).
+{{</note>}}
+
+### Cloning in YSQL
+
+Because YugabyteDB is PostgreSQL compatible, you can create a database as a clone of another using the `TEMPLATE` SQL option of `CREATE DATABASE` command as follows:
+
+```sql
+CREATE DATABASE clone_db TEMPLATE original_db;
+```
+
+`clone_db` is created as a clone of database `original_db` which contains the latest schema and data of `original_db` as of current time.
+Additionally, to create a clone of the original database at a specific point in time in the past (within the history retention period specified when creating the snapshot schedule) you can specify the [Unix timestamp](https://www.unixtimestamp.com/) in microseconds in the `AS OF` option as follows:
+
+```sql
+CREATE DATABASE clone_db TEMPLATE origginal_db AS OF 1723146703674480;
+```
+
+### Cloning in YCQL
+
+You can create a clone in YCQL using the yb-admin command as follows:
+
+```sh
+./bin/yb-admin --master_addresses $MASTERS clone_namespace ycql.originaldb1 clonedb2 1715275616599020
+```
+
+`clonedb2` is created as as a clone of the ycql database `originaldb1` as of 1715275616599020 Unix timestamp.
+
+### Check the clone status
+
+You can check the status of clone operations performed on a database with the `source_database_id` (YSQL)/`source_namespace_id`(YCQL) field using the yb-admin command `list_clones` as follows:
+
+```sh
+./bin/yb-admin --master_addresses $MASTERS list_clones 00004000000030008000000000000000
+```
+
+```output
+[
+    {
+        "aggregate_state": "COMPLETE",
+        "source_namespace_id": "00004000000030008000000000000000",
+        "seq_no": "1",
+        "target_namespace_name": "testing_clone_db",
+        "restore_time": "2024-08-09 21:42:16.451974"
+    },
+    {
+        "aggregate_state": "COMPLETE",
+        "source_namespace_id": "00004000000030008000000000000000",
+        "seq_no": "2",
+        "target_namespace_name": "dev_clone_db",
+        "restore_time": "2024-08-09 21:42:55.048663"
+    }
+]
+```
+
+Note that there were two clone operations performed on the source database `00004000000030008000000000000000` that are COMPLETE. The two clones are `testing_clone_db` and `dev_clone_db` and they each have a unique `seq_no` used to identify each clone operation from the same source database.
+
+Alternatively, you can check the status of a specific clone operation if you have both the `source_database_id` (YSQL)/`source_namespace_id`(YCQL) and the `seq_no` as follows:
+
+```sh
+./bin/yb-admin --master_addresses $MASTERS list_clones 00004000000030008000000000000000 2
+```
+
+```output
```

Comment:
The output doesn't look any different what changed

Reply:
my bad. had to include the output only with seq_no=2. Updated it.